api.yaml

```diff
@@ -38,18 +52,22 @@ paths:
                 $ref: '#/components/examplesImport/complexeImportLead'
         required: true
       responses:
-        '201':
-          description: Successful operation
-        '422':
-          description: Invalid input
-        '409':
-          description: Duplicate value
+        201 : 
+          description : The import was **successful**. *You need to make sure that nobody will  communicate with the lead while it is in by Pinpo’s hands. Two potential ideas to explore, make the object invisible in the CRM or assign the object to a Pinpo user in the CRM.*
+        409 :
+          description : Duplicate value.
```

Comment:
```suggestion
          description : The import **is not successful** because of duplicated value. This lead has been already imported in our platform and can't be proceed twice.
```